tests/background.py

```diff
@@ -361,7 +361,76 @@ def testCFHT_oldAPI(self):
         if display:
             ds9.mtv(mi, frame = 1)
 
-            
+    def testXY0(self):
+        """Test fitting the background to an image with nonzero xy0
+
+        The statsImage and background image should not vary with xy0
+        """
+        bgImageList = [] # list of background images, one per xy0
+        statsImageList = [] # list of stats images, one per xy0
+        for xy0 in (afwGeom.Point2I(0, 0), afwGeom.Point2I(1000, 500)):
+            afwdataDir = eups.productDir("afwdata")
+            if not afwdataDir:
+                print >> sys.stderr, "Skipping testCFHT as afwdata is not setup"
+                return
+
+            mi = afwImage.MaskedImageF(os.path.join(afwdataDir,
```

Comment:
This and the next few lines are near-duplicates with `testSubImage`. If there was much more duplication, I'd strongly urge you to consider refactoring them into a function which both these tests can call. Since it's only a few lines, I will just suggest that as something you might like to consider.